src/datta_lab_to_nwb/markowitz_gillis_nature_2023/fiberphotometryinterface.py

```diff
@@ -0,0 +1,189 @@
+            coordinates=(0.260, 2.550, -2.40),  # (AP, ML, DV)
+        )
+
+        # Fibers Table
+        fibers_table = FibersTable(
+            description="FibersTable",
+        )
+        nwbfile.add_lab_meta_data(
+            FiberPhotometry(
+                fibers=fibers_table,
+                excitation_sources=excitation_sources_table,
+                photodetectors=photodetectors_table,
+                fluorophores=fluorophores_table,
+            )
+        )
+
+        # Important: we add the fibers to the fibers table _after_ adding the metadata
+        # This ensures that we can find this data in their tables of origin
+        fibers_table.add_fiber(
+            excitation_source=0,  # integers indicated rows of excitation sources table
+            photodetector=0,
+            fluorophores=[0],  # potentially multiple fluorophores, so list of indices
+            location=metadata["FiberPhotometry"]["area"],
+            notes="None",
+        )
+        fibers_table.add_fiber(
+            excitation_source=1,  # integers indicated rows of excitation sources table
+            photodetector=0,
+            fluorophores=[0],  # potentially multiple fluorophores, so list of indices
+            location=metadata["FiberPhotometry"]["area"],
+            notes="None",
+        )
+
+        # ROI Response Series
+        # Here we set up a list of fibers that our recording came from
+        fibers_ref = DynamicTableRegion(
+            name="rois", data=[0, 1], description="source fibers", table=fibers_table  # potentially multiple fibers
+        )
+
+        signal_series = RoiResponseSeries(
+            name="signal_dff",
+            description="signal dF over F",
+            data=session_df.signal_dff.to_numpy(),
+            unit="dF/F",
+            timestamps=session_df.timestamp.to_numpy(),
+            rois=fibers_ref,
+        )
+        reference_series = RoiResponseSeries(
+            name="reference_dff",
+            description="reference dF over F",
+            data=session_df.reference_dff.to_numpy(),
+            unit="dF/F",
+            timestamps=signal_series.timestamps,
+            rois=fibers_ref,
+        )
+        reference_fit_series = RoiResponseSeries(
+            name="reference_dff_fit",
+            description="reference fit dF over F",
+            data=session_df.reference_dff_fit.to_numpy(),
+            unit="dF/F",
+            timestamps=signal_series.timestamps,
+            rois=fibers_ref,
+        )
+        uv_reference_fit_series = RoiResponseSeries(
+            name="uv_reference_fit",
+            description="uv reference F",
+            data=session_df.uv_reference_fit.to_numpy(),
+            unit="F",
+            timestamps=signal_series.timestamps,
+            rois=fibers_ref,
+        )
+
+        # Aggregate into OPhys Module
+        ophys_module = nwbfile.create_processing_module(name="ophys", description="fiber photometry")
```

Comment:
Recommend `get_module` here for future reususability of the interface

Reply:
whoops, forgot about that 😅